cs-nutanix-prism/src/main/java/io/cloudslang/content/nutanix/prism/utils/Constants.java

```diff
@@ -140,14 +141,12 @@
         public static final String SUCCEEDED = "Succeeded";
         public static final String FAILED = "Failed";
         public static final String TASK_FAILURE_PATH = "$.meta_response.error_detail";
-        public static final String TASK_UUID_PATH = "uuid";
```

Comment:
Change to TASK_UUID_PATH = "task_uuid"

Reply:
This line is deleted. Not needed for this operation.